highlighting/definition.py

```diff
@@ -66,15 +66,21 @@
 
 
 TOKEN_CONSTANT = "text"
+# 
+# 
+def translate(word, withoutGroup = False):
+	""" Function that allows to add tags around the keywords to be translated.
```

Comment:
tags is `__`? (called dunderscore in Python)?

Reply:
yes !